### DISCUSSION TITLE
Introduce new label for stable releases of ubuntu-latest

### DISCUSSION THREAD

**Discussion by ksaffarian** [actions/virtual-environments#6613] [Image configuration]
Hi

Recently, with the publish of `ubuntu-latest` release **20221111.1**, few breaking changes slipped through and impacted customers. One of those was new PowerShell 7.3 feature called **PSNativeCommandArgumentPassing** (https://github.com/actions/runner-images/issues/6586).

I want to suggest that we use `ubuntu-latest` on quarterly releases or maybe use a new label called `ubuntu-stable` for those that need more time to adjust their pipelines to new breaking changes.

This also give us enough time for Azure DevOps team to get feedback from the field regarding new releases.

I understand that we can use private agents with custom image that we maintain ourselves. However, I think the current release cycles of `ubuntu-latest` is too quick and the impact is considerable.

What do you think?

Thanks

**Comment by mikhailkoliada** [comment]
@ksaffarian we generally think of stable releases at the moment, it is possibly going to be improved in the future.
